readme.md

```diff
@@ -0,0 +1,68 @@
+<snippet>
+<content>
+
+[![Build Status](https://travis-ci.org/andela-hoyeboade/amity_room_allocator.svg?branch=working)](https://travis-ci.org/andela-hoyeboade/amity_room_allocator) [![Coverage Status](https://coveralls.io/repos/github/andela-hoyeboade/amity_room_allocator/badge.svg?branch=working)](https://coveralls.io/github/andela-hoyeboade/amity_room_allocator?branch=working)
+
+# Amity Room Allocator
+Python console application for allocating fellows and staffs to offices or living spaces in Amity.
+
+## Installation
+Clone the repo
+```git clone https://github.com/andela-hoyeboade/amity_room_allocator.git/``` and navigate to the project directory
+
+Install dependencies
+```pip install -r requirements.txt```
+
+Run the program 
+```python amity.py``` shows a list of available commands
+
+## Usage:
+```
+  amity.py create_room (<room_names> <room_types>)...
+  amity.py add_person <first_name> <last_name> <STAFF/FELLOW> [<wants_accommodation>]
+  amity.py reallocate_person <person_identifier> <new_room_name>
+  amity.py load_people <filename>
+  amity.py print_room <room_name>
+  amity.py print_allocations [--o=filename]
+  amity.py print_unallocated [--o=filename]
+  amity.py save_state [--db=sqlite_database]
+  amity.py load_state [--db=sqlite_database]
+  amity.py print_people
+  amity.py -h | --help
+```
+## Usage Examples:
+
+  ```python amity.py create_room Moon office``` Create a new office Moon <br />
```

Comment:
**README:** Change the placement, here my suggestion:
Create a new office called _Moon_:
```
python amity.py create_room Moon office
```